### PR TITLE
[#327] Introduce a release script that automates the process

### PR DESCRIPTION
**Pull request by paulodiniz**
Fixes https://github.com/airyhq/airy/issues/327

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    command git push origin release/${release_number}
+}
+
+finish() {
+    release_number=$1
+    echo -e "Finishing release ${release_number}"
+    increase_version
+    commit_version
+    merge_main
+    merge_develop
```
Comment (paulodiniz) [inline scripts/release.sh:32]: When do we close the issue?

**Review by ljupcovangelski — COMMENTED** [review]
I think we need couple of things more at this stage:
- check for the variables if they are there ($1, $2) and what are their values, so that we don't get any unexpected behavior
- short help/usage message, so that we see how to invoke and run the release

We can give it a proper test as soon as next Tuesday I guess. The others things we can add along the way.

File: scripts/release.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```
Comment (lucapette) [inline scripts/release.sh:1]: we have this "unofficial bash script mode" in other scripts and we should add it here as well

File: scripts/release.sh (hunk 1 above)
Comment (lucapette) [reply scripts/release.sh:32]: it should close automatically based on the commit message?

**Review by lucapette — COMMENTED** [review]
First quick pass! Looking great already :)

File: scripts/release.sh [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+merge_main() {
+    command git checkout main
+    command git pull origin main
+    command git merge --no-ff release/${release_number}
+    command git tag ${release_number}
+    command git push origin main
+    command git push origin ${release_number}
+    echo -e "Sucesfully merged into main branch\n"
```
Comment (lucapette) [inline scripts/release.sh:62]:
```suggestion
    echo -e "Sucessfully merged into main branch\n"
```

Comment (lucapette) [inline scripts/release.sh:62]:
```suggestion
    echo -e "Successfully merged into main branch\n"
```

File: scripts/release.sh [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+merge_develop() {
+    command git checkout develop
+    command git pull origin develop
+    command git merge --no-ff release/${release_number}
+    command git push origin develop
+    echo -e "Sucesfully merged into develop branch\n"
```
Comment (lucapette) [inline scripts/release.sh:70]:
```suggestion
    echo -e "Successfully merged into develop branch\n"
```

**Review by pascal-airy — APPROVED** [review]

**Pull request merged**